CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - On macOS, fix the mouse buttons other than left/right/middle being reported as middle.
 - On Wayland, support fractional scaling via the wp-fractional-scale protocol.
 - On web, fix removal of mouse event listeners from the global object upon window distruction.
+- Added support for setting the IME purpose, currently implemented on Wayland only.
```

Comment:
Could you write out the exact API that you've added? Like `Window::set_ime_purpose`?